openedx/core/djangoapps/notifications/views.py

```diff
@@ -286,9 +284,20 @@ def get(self, request):
 
             count_total += count
             count_by_app_name_dict[app_name] = count
-        # Return the unseen notifications count for the user and the unseen notifications count for each app name.
 
+        # Retrieve active course enrollments for the current user
+        learner_enrollments = CourseEnrollment.objects.filter(user=request.user, is_active=True)
+
+        # Check if the show_notifications_tray flag is enabled for any of the learner's enrollments
```

Comment:
Instead of adding inline comments. It is better to explain what the method is doing in the doc string. Inline comments should only be added if the code is not self explanatory.